.github/workflows/android.yml

```diff
@@ -0,0 +1,22 @@
+name:  Android build CI
+
+on: pull_request 
+
+jobs:
+ build-app:
+   runs-on: ubuntu-latest
+   steps:
+     - name: Set up JDK
+       uses: actions/setup-java@v1
+       with:
+         java-version: 1.8
+     - name: run Tests
+       run: |
+          cd TrailGage
+          ./gradlew test --info
+          
+     - name: Build App
+       run: |
+          cd TrailGage
+          ./gradlew build --stacktrace
```

Comment:
Before you can `cd TrailGage` you need to clone/checkout your repository! The action will not do that automatically for you.